server/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestGetSnapshotsAction.java

```diff
@@ -53,6 +53,19 @@ public RestChannelConsumer prepareRequest(final RestRequest request, final NodeC
         GetSnapshotsRequest getSnapshotsRequest = getSnapshotsRequest(repositories).snapshots(snapshots);
         getSnapshotsRequest.ignoreUnavailable(request.paramAsBoolean("ignore_unavailable", getSnapshotsRequest.ignoreUnavailable()));
         getSnapshotsRequest.verbose(request.paramAsBoolean("verbose", getSnapshotsRequest.verbose()));
+        final GetSnapshotsRequest.SortBy sort = GetSnapshotsRequest.SortBy.of(request.param("sort", getSnapshotsRequest.sort().toString()));
+        final int size = request.paramAsInt("size", getSnapshotsRequest.size());
+        final String[] afterString = request.paramAsStringArray("after", Strings.EMPTY_ARRAY);
```

Comment:
Fortunately not :) We validate snapshot names for:

```
        if (snapshotName.contains(",")) {
            throw new InvalidSnapshotNameException(repositoryName, snapshotName, "must not contain ','");
        }
```

We can't have `,` in names because we have APIs like this one that allow fetching snapshots by names comma separated :)